.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           - ubuntu-latest
           - windows-latest
         ocaml-compiler:
-          - 4.10.2
+          - 4.12.1
```

Comment:
@zbaylin Was this to work around the Windows issues? I'd rather move the update to newer compiler to a separate PR if possible (I think we should prob add some kind of cross-version builds + tests in ci, but that's a larger / different topic).

Reply:
Yeah that was to see if the OCaml version was causing the CI failure -- I'll change it back now :)

Reply:
Reverted in 95db987 😃